sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedOptions.cs

```diff
@@ -34,23 +36,33 @@ internal DataFeedOptions(DataFeedDetail dataFeedDetail)
         }
 
         /// <summary>
+        /// A description of the <see cref="DataFeed"/>.
         /// </summary>
         public string FeedDescription { get; set; }
 
+        // TODODOCS.
         /// <summary>
         /// </summary>
         public AccessMode? AccessMode { get; set; }
 
         /// <summary>
+        /// Configures the behavior of this <see cref="DataFeed"/> for rolling-up the ingested data
+        /// before detecting anomalies.
         /// </summary>
         public DataFeedRollupSettings RollupSettings { get; set; }
 
         /// <summary>
+        /// Configures the behavior of this <see cref="DataFeed"/> when dealing with missing points
+        /// in the data ingested from the data source.
         /// </summary>
         public DataFeedMissingDataPointFillSettings MissingDataPointFillSettings { get; set; }
 
         /// <summary>
+        /// The emails of this data feed's administrators. Administrators have total control over a
+        /// data feed, being allowed to update, delete or pause them. They also have access to the
+        /// credentials used to authenticate to the data source.
         /// </summary>
+        /// <exception cref="ArgumentNullException"><see cref="Administrators"/> is null.</exception>
```

Comment:
where do people set this behavior? Do you think adding a link with documentation for that will help?

Reply:
I'm not sure if I'm following. This property has a setter that can be used to specify admins.

Reply:
I think I am confused. So the idea is that putting someone's name here the service gives them admin access to all info?

Reply:
The way I read it is as defaults. i.e Here is the list of people that have admin permissions, instead of here is the list, and if you add/modify, they get admin permissions too

Reply:
> So the idea is that putting someone's name here the service gives them admin access to all info?

Yes.

> Here is the list of people that have admin permissions, instead of here is the list, and if you add/modify, they get admin permissions too

Why do you feel like that towards this property in particular (maybe the name?)? It lives inside the `DataFeedOptions` class, with other properties that are settable and modifiable.